test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsing.cs

```diff
@@ -23,6 +23,11 @@ public class RequestParsing
 
         private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
 
+        private const string plaintextTechEmpower = "GET /plaintext HTTP/1.1\r\n" +
```

Comment:
I merged it before seeing these comments 😛 

I'll make a smaller change to switch the baseline to tech empower, before merging https://github.com/aspnet/KestrelHttpServer/pull/1426.